cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/cluster-settings-step/ClusterSettings.scss

```diff
@@ -0,0 +1,38 @@
+.cluster-settings-container {
```

Comment:
Since we will also be dealing with workload clusters, maybe naming the file ManagementClusterSettings, and calling this class .management-cluster-settings-container, etc? I'm not sure if we'll use the same CSS classes for workload clusters

Reply:
css issues will be handled in another PR.